app/views/layouts/_cart-dropdown.html.erb

```diff
@@ -0,0 +1,38 @@
+<% if cart && (cart.user_cart_items.count > 0) %>
+  <li style="width: 450px;">
+    <table class="table">
+      <tr>
+        <th>Preview</th>
+        <th>Item</th>
+        <th>Amount</th>
+        <th>Total</th>
+        <th>Actions</th>
+      </th>
+      <% cart.user_cart_items.each do |item| %>
+        <% if item.id %>
+          <tr>
+            <td>
+              <% if item.variation.product.photo %>
+                <%= image_tag item.variation.product.photo.url, class: 'cart-image'  %>
+              <% end %>
+            </td>
+            <td>
+              <%= item.variation.product.name %>
+              <br />( <%= item.variation.label %> )
+            </td>
+            <td> <%= item.amount %> </td>
+            <td>$<%= item.amount*item.variation.price %></td>
+            <td><%= link_to 'Remove', shop_cart_path(item: item), method: :delete %></td>
+          </tr>
+        <% end %>
+      <% end %>
+    </table>
+  </li>
+<% else %>
+  <li style="width: 300px;"> Cart is empty.. </li>
+<% end %>
+<li>
+  <div id="viewcart">
```

Comment:
Are we doing anything specific to this div assigned to #viewcart? Or is it just labelled for segregation?